deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml

```diff
@@ -84,7 +84,7 @@ spec:
             - name: socket-dir
               mountPath: /csi
         - name: csi-snapshotter
-          image: k8s.gcr.io/sig-storage/csi-snapshotter:v4.2.1
+          image: gcr.io/k8s-staging-sig-storage/csi-snapshotter:v5.0.0-rc1
```

Comment:
Can you add a TODO here to replace this with the official image once v5.0.0 is released?

Reply:
Done